hack/make-rules/run-e2e-tests.sh

```diff
@@ -25,20 +25,112 @@ source "${YURT_ROOT}/hack/lib/build.sh"
 readonly LOCAL_ARCH=$(go env GOHOSTARCH)
 readonly LOCAL_OS=$(go env GOHOSTOS)
 readonly YURT_E2E_TARGETS="test/e2e/yurt-e2e-test"
+cloudNodeContainerName="openyurt-e2e-test-control-plane"
+edgeNodeContainerName="openyurt-e2e-test-worker"
+edgeNodeContainer2Name="openyurt-e2e-test-worker2"
 KUBECONFIG=${KUBECONFIG:-${HOME}/.kube/config}
 
+
+function set_flags() {
+    goldflags="${GOLDFLAGS:--s -w $(project_info)}"
+    gcflags="${GOGCFLAGS:-}"
+    goflags=${GOFLAGS:-}
+
+#    set kubeconfig
+    docker exec -d $edgeNodeContainerName /bin/bash -c 'mkdir /root/.kube/ -p'
+    docker exec -d $edgeNodeContainerName /bin/bash -c  "echo 'export KUBECONFIG=/root/.kube/config' >> /etc/profile && source /etc/profile" 
+    docker cp $KUBECONFIG $edgeNodeContainerName:/root/.kube/config
+}
+
+# set up flannel
+function set_up_flannel() {
+    local flannelYaml="https://raw.githubusercontent.com/flannel-io/flannel/master/Documentation/kube-flannel.yml"
+    local flannelDs="kube-flannel-ds"
+    local flannelNameSpace="kube-flannel"
+    local POD_CREATE_TIMEOUT=120s
+    curl -o /tmp/flannel.yaml $flannelYaml
+    kubectl apply -f /tmp/flannel.yaml
+    # check if flannel on every node is ready, if so, "daemon set "kube-flannel-ds" successfully rolled out"
+    kubectl rollout status daemonset kube-flannel-ds -n kube-flannel --timeout=${POD_CREATE_TIMEOUT}
+
+    # set up bridge cni plugins for every node
+    wget -O /tmp/cni.tgz https://github.com/containernetworking/plugins/releases/download/v1.1.1/cni-plugins-linux-amd64-v1.1.1.tgz
+
+    docker cp /tmp/cni.tgz $cloudNodeContainerName:/opt/cni/bin/
+    docker exec -t $cloudNodeContainerName /bin/bash -c 'cd /opt/cni/bin && tar -zxf cni.tgz'
+
+    docker cp /tmp/cni.tgz $edgeNodeContainerName:/opt/cni/bin/
+    docker exec -t $edgeNodeContainerName /bin/bash -c 'cd /opt/cni/bin && tar -zxf cni.tgz'
+
+    docker cp /tmp/cni.tgz $edgeNodeContainer2Name:/opt/cni/bin/
+    docker exec -t $edgeNodeContainer2Name /bin/bash -c 'cd /opt/cni/bin && tar -zxf cni.tgz'    
+}
+
+# install gingko
+function get_ginkgo() {
+    go install github.com/onsi/ginkgo/v2/ginkgo@v2.1.4
+}
+
 # run e2e tests
-function run_e2e_tests {
+function run_non_edge_autonomy_e2e_tests {
     # check kubeconfig
     if [ ! -f "${KUBECONFIG}" ]; then
         echo "kubeconfig does not exist at ${KUBECONFIG}"
         exit -1
     fi
+    # run non-edge-autonomy-e2e-tests
+    cd $YURT_ROOT/test/e2e/
+    ginkgo --gcflags "${gcflags:-}" ${goflags} --ldflags "${goldflags}" --label-filter='!edge-autonomy' -r
```

Comment:
How about adding `-v` for more verbose output

Reply:
will do